packages/hoppscotch-sh-admin/src/components/settings/DataSharing.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="md:grid md:grid-cols-3 md:gap-4 border-divider border-b py-8">
+    <div class="px-8 md:col-span-1">
+      <h3 class="heading">{{ t('configs.data_sharing.title') }}</h3>
+      <p class="my-1 text-secondaryLight">
+        {{ t('configs.data_sharing.description') }}
+      </p>
+    </div>
+
+    <div class="mt-5 mx-8 md:col-span-2">
```

Comment:
```suggestion
    <div class="mx-8 md:col-span-2">
```
we can align the left and right section by removing the top margin